google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -468,10 +469,10 @@ public Builder setCredentialsProvider(CredentialsProvider credentialsProvider) {
     }
 
     /**
-     * Gives the ability to set a custom executor for managing lease extensions. If none is provided
-     * a shared one will be used by all {@link Subscriber} instances.
+     * Gives the ability to set a custom executor for polling and managing lease extensions. If none
+     * is provided a shared one will be used by all {@link Subscriber} instances.
      */
-    public Builder setLeaseAlarmsExecutorProvider(ExecutorProvider executorProvider) {
+    public Builder setSystemExecutorProvider(ExecutorProvider executorProvider) {
```

Comment:
I agree that this should be renamed from "LeaseAlarms," but I don't know if "System" is the right term here.